internal/build.go

```diff
@@ -159,6 +159,38 @@ func assertRuleIdsWithoutSnykPrefix(rules []util.Rule) error {
 	return nil
 }
 
+func assertValidSeverityLevel(rules []util.Rule) error {
+	invalidRulesPaths := []string{}
+
+	for _, rule := range rules {
+		var isValidSeverityLevel = func(severityLevel string) bool {
```

Comment:
nit: This is just my opinion, but I couldn't see the benefit of having an inline anonymous function stored in a named variable here, instead of declaring it as a regular function. 

There's no usage of closure here, and it's still technically a named function, not an IIFE. 
I'm not sure how GoLang's compilation handles these functions, so I can't say how this affects performance.